.github/workflows/publish_docs.yaml

```diff
@@ -0,0 +1,61 @@
+name: Publish documentation
+
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    if: "!contains(github.event.head_commit.message, '[skip ci]') && !contains(github.event.head_commit.message, '[ci skip]')"
+
+    steps:
+      - uses: actions/checkout@v2
+        name: Check out source-code repository
+
+      - name: Setup Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.9
+
+      - name: Install pip
+        run: python -m pip install --upgrade pip
+
+      # Required to automatically extract Click CLI and Flask API documentation
+      - name: Install MegaQC
+        run: pip install -e .[dev]
```

Comment:
Ah this is why it didn't work for me - the GHA code seems to be doing the right thing here and installing dev ;-)

Reply:
Yeah, forgot to mention that :)